### PR TITLE
Support loading Starlark modules from Git

### PR DESCRIPTION
**Pull request by edigaryev**
See #53.

Some things that still need to be taken care of:

* make Git-related storages bounded to a specific constant (e.g. 8 MiB)
https://github.com/cirruslabs/cirrus-cli/blob/04bbcae3addfcd5007460a26fad2e7353f40523d/pkg/larker/loader/git/git.go#L65-L66
* write a few more tests

### CODE REVIEW

File: pkg/larker/loader/git/git.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	if err := worktree.Checkout(&git.CheckoutOptions{
+		Hash: *hash,
+	}); err != nil {
+		return nil, err
+	}
+
+	// Read the file from the working tree
+	file, err := worktree.Filesystem.Open(locator.Path)
+	if err != nil {
+		return nil, err
```
Comment (fkorotkov) [inline pkg/larker/loader/git/git.go:97]: Will it have a pretty error message?

**Review by fkorotkov — COMMENTED** [review]
I think it will be good to add a test or two about error messages when people will make typos like `.start` instead of `.star`. Otherwise LGTM so far.

**Comment by edigaryev** [comment]
>make Git-related storages bounded to a specific constant (e.g. 8 MiB)

Implemented in https://github.com/cirruslabs/cirrus-cli/pull/85/commits/4e9fd367278d63c0d8685c2364bf2df9d1203f8b.

Unfortunately, the accounting is quite imprecise due to details hidden in `github.com/go-git/go-billy/v5/memfs` implementation. The most evident shortcomings of this are:

* we don't count intermediate files created as a result of calling `MkdirAll()` towards a limit
* we count all written bytes towards a limit and ignore the fact that `Write()`'s can overwrite the same file piece without increasing it's size

If at some point we want these to be fixed too, the next obvious step would be to submit the modifications to the https://github.com/go-git/go-billy itself.

**Comment by fkorotkov** [comment]
Another option might be to store to a temp directory on disk without bounds and then delete it. Let's think about it more when implementing the microservice and caching of the 'load' results.

File: pkg/larker/loader/git/git.go (hunk 1 above)
Comment (edigaryev) [reply pkg/larker/loader/git/git.go]: See https://github.com/cirruslabs/cirrus-cli/pull/85/commits/cb79e3f39324792108e0741337ae800754cc4a39.

**Comment by edigaryev** [comment]
> I think it will be good to add a test or two about error messages when people will make typos like `.start` instead of `.star`.

See https://github.com/cirruslabs/cirrus-cli/pull/85/commits/f8924a96f68d2dbeb96d396d130726345d1fdf87.

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**